roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -124,8 +124,12 @@ rabbit_password={{ secrets.rabbit_password }}
 # Reserved Resources
 {% if ironic.enabled -%}
 reserved_host_memory_mb=0
-{% elif inventory_hostname in groups['controller'] -%}
-reserved_host_memory_mb=4096
+{% elif nova.reserved_host_memory_mb is defined %}
```

Comment:
Where would this be defined? I don't see it above. Is this just for a blanket reservation regardless of control or compute?